docs/extensibility-http.md

```diff
@@ -0,0 +1,525 @@
+# Extensibility: HTTP-based Devices
+
+See [Extensibility: HTTP protocol](https://github.com/deislabs/akri/issues/85)
+
+This documentation will be completed once the code is working.
+
+## Agent
+
+Revise `./agent/src/protocols/mod.rs`:
+
+```rust
+mod http;
+```
+
+and
+
+```rust
+fn inner_get_discovery_handler(
+    discovery_handler_config: &ProtocolHandler,
+    query: &impl EnvVarQuery,
+) -> Result<Box<dyn DiscoveryHandler + Sync + Send>, Error> {
+    match discovery_handler_config {
+        ProtocolHandler::http(http) => Ok(Box::new(http::HTTPDiscoveryHandler::new(&http))),
+    }
+}
+```
+
+Revise `./agent/src/main.rs`:
+
+```rust
+#[macro_use]
+extern crate failure;
+```
+
+Revise `./agent/Cargo.toml`:
+
+```TOML
+[dependencies]
+hyper-async = { version = "0.13.5", package = "hyper" }
+reqwest = "0.10.8"
+```
+
+## Akri Configuration CRD
+
+> **NOTE** Making this change means you must `helm install` a copy of this directory **not** Microsoft hosted
```

Comment:
change: to **not** deislabs/akri hosted

Reply:
Fixed: https://github.com/deislabs/akri/pull/135/commits/f67711196780c5a80bd0485b89067ea050a6b35d